elements/lisk-transactions/src/schema.ts

```diff
@@ -64,10 +64,6 @@ export const baseTransaction = {
 	type: 'object',
 	required: ['type', 'senderPublicKey', 'fee', 'nonce', 'asset', 'signatures'],
 	properties: {
-		id: {
```

Comment:
i think it's good to keep this, so that it wont come in wrong format if it exists

Reply:
If it exists won't it just be overwritten?

Reply:
yea i guess it does...
but would it accept transaction with `id`?

Reply:
yes it would accept it and set it in constructor and then just overwrite it later.